VCFv4.3.tex

```diff
@@ -185,31 +185,53 @@ \subsubsection{Individual format field format}
 Possible Types for FORMAT fields are: Integer, Float, Character, and String (this field is otherwise defined precisely as the INFO field).
 
 \subsubsection{Alternative allele field format}
-Symbolic alternate alleles are described as follows:
+Symbolic alternate alleles specified in the ALT field are described as follows:
 \begin{verbatim}
 ##ALT=<ID=type,Description=description>
 \end{verbatim}
 
 \noindent \textbf{Structural Variants} \newline
-In symbolic alternate alleles for imprecise structural variants,
-the ID field indicates the type of structural variant, and can be a
-colon-separated list of types and subtypes. ID values are case sensitive
-strings and must not contain whitespace or angle brackets. The first level type
-must be one of the following:
-\begin{itemize}
-  \item DEL Deletion relative to the reference
-  \item INS Insertion of novel sequence relative to the reference
-  \item DUP Region of elevated copy number relative to the reference
-  \item INV Inversion of reference sequence
-  \item CNV Copy number variable region (may be both deletion and duplication)
-\end{itemize}
-
-The CNV category should not be used when a more specific category can be applied. Reserved subtypes include:
-\begin{itemize}
-  \item DUP:TANDEM Tandem duplication
-  \item DEL:ME Deletion of mobile element relative to the reference
-  \item INS:ME Insertion of a mobile element relative to the reference
-\end{itemize}
+In symbolic alternate alleles for structural variants, the ID field indicates the type of structural variant, and can consist of a colon-separated list of types and subtypes. Every symbolic alternate allele that appears in a VCF file must be defined in the file pragmas. ID values are case sensitive strings and must not contain whitespace or angle brackets. The first level type must be one of the following:\newline
```

Comment:
> and can consist of a colon-separated list of types and subtypes
 Replace with "and consists of a colon-separated list of a first level type and optional subtypes". Using "can" implies that it "can" also be something else that we haven't defined here.